docs/api_definitions/custom_plugin_python/custom_plugin_python.md

```diff
@@ -0,0 +1,73 @@
+## Custom Plugin example using python
+
+We assume that you already have a developed python plugin with the associated python file and its manifest in JSON.
+If not, you can start from the provided examples ("manifest.json" and "middleware.py").
+Furthermore, in this example, we are going to serve those as a bundle using a small http python server.
+
+
+Firstly to create the said bundle we need to run the following command:
+
+```
+IMAGETAG=v3.2.1
+
+docker run \
+  --rm -w "/tmp" -v $(pwd):/tmp \
+  --entrypoint "/bin/sh" -it \
+  tykio/tyk-gateway:$IMAGETAG \
+  -c '/opt/tyk-gateway/tyk bundle build -y'
+
+```
+
+We are setting the shell variable IMAGETAG to be the version of the gateway we intend on loading the python bundle onto. 
+In this case, we are loading the plugin onto a Tyk gateway v3.2.1. When completed, you should see a bundle.zip in your current directory.
+
+Now we must serve the bundle, and the easiest way to do so is through a local server using the following command:
+``` 
+python3 -m http.server
+```
+
+Your bundle should now be accessible locally through the URL http://localhost:8000/bundle.zip,
+or through the url http://host.minikube.internal:8000/bundle.zip from within minikube.
```

Comment:
It may sound silly but why are we mentioning minikube here? The above part indicates that I should see `bundle.zip in my current directory`. And after serving it, shouldn't it be accessible only locally? It seems like shouldn't be accessible via my cluster. Maybe i am missing a feature of minikube 😄

Reply:
Yes absolutely, I can add some more information to the markdown file

Reply:
Thank you 🙏

Reply:
I'm also a bit confused about this. Targeting bundle server http://host.minikube.internal:8000/bundle.zip might work fine locally / in Minikube, but how would it work in production? I need a bundle server right? So that bundle server needs to be deployable to K8s also with a Deployment and Service that the Gateway can target?